src/main/java/core/basesyntax/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements MyFileReader {
+    private List<String> inputReport = new ArrayList<>();
```

Comment:
This field should be local to the read method. Since it's being reassigned every time the method is called, it doesn't need to be a class-level variable.